### ISSUE TITLE
Persistent IDs should be consistent and integers across model parameterizations

### ISSUE THREAD

**Issue by smjenness** [statnet/EpiModel#307]
here's the current storage for bipartite networks:

```
head(get_transmat(sim1, sim = 1), 10)
   at         sus         inf infDur transProb actRate finalProb
1   2 5ff4952cb01 5ff373df9b6     27       0.1       5   0.40951
2   2 5ff1dfbdb3c 5ff5278d95b     28       0.1       5   0.40951
3   2 5ff5831c407 5ff7f48d4b9      7       0.1       5   0.40951
4   2 5ff5f80368a 5ff3153849a     24       0.1       5   0.40951
5   2 5ff6fc977ef 5ff3153849a     24       0.1       5   0.40951
6   2  5ffcda6ab1 5ff4a407a09     40       0.1       5   0.40951
7   3 5ff5831c407 5ff7f48d4b9      8       0.1       5   0.40951
8   3 5ff6cf91e6c   5ff681a44     50       0.1       5   0.40951
9   4 5ff137eb1db 5ff4952cb01      2       0.1       5   0.40951
10  4 5ff12fe42b4 5ff110cca80     10       0.1       5   0.40951
```

Just use integers instead, similar to EpiModelHIV's `uid`.

**Comment by skyebend** [comment]
If these were generated using the network dynamic persistant id functionality, i think you can instead provide ids, or tell it to use a pre-existing vertex attribute (even vertex.names, if they are already unique integers)

**Comment by smjenness** [comment]
Closed in #354
